etl/steps/data/garden/survey/2023-08-04/trust_surveys.meta.yml

```diff
@@ -0,0 +1,46 @@
+# NOTE: To learn more about the fields, hover over their names.
```

Comment:
Hey @pabloarosado @Marigold ! How should I set this up to run without errors? I am concatenating three different tables and I get:
AssertionError: Column `trust` must have either sources or origins

I saw sources is deprecated and origins is not in the schema for the yaml, so how should I proceed?

Reply:
@paarriagadap the problem is that origins are not propagated properly. You should be using `pr.read_csv` and `pr.merge` instead of their `pd.` versions. See my other comments.

Reply:
Bah, right, thanks. I tried to adapt this old code from August and I forgot that. I had only changed the metadata and parts of the code.